test/testdata/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: template-service-broker
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
```

Comment:
worried about the how this gets productised (something I know nothing about) - is it down to ansible and oc cluster up to handle setting this according to the version and handling upgrade? presumably oc cluster up should be setting this then?